src/rsa/signing.rs

```diff
@@ -219,6 +217,20 @@ impl RSAKeyPair {
     pub fn public_modulus_len(&self) -> usize {
         self.n_bits.as_usize_bytes_rounded_up()
     }
+
+    /// Exports the public modulus as big endian to the output array.
+    ///
+    /// The array should be of at least the size returned by
+    /// `public_modulus_len`.
+    pub fn export_public_modulus(&self, out: &mut [u8])
+                    -> Result<(), error::Unspecified> {
+        self.n.fill_be_bytes(out)
```

Comment:
Shouldn't we leave this to the user? The user must allocate enough space somehow based on `public_modulus_len`. If there's not enough space, `fill_be_bytes` errors, if there's more space, it's padded, but that may be the user's intent. Doesn't `verify_rsa` also accept padded input?

I'll look into the unit tests!